recipes/hhsuite/meta.yaml

```diff
@@ -51,3 +51,4 @@ extra:
     - milot-mirdita
   additional-platforms:
     - linux-aarch64
+    - osx-arm
```

Comment:
@martin-g Should this not be `osx-arm64`? Attempting raising PR here: https://github.com/bioconda/bioconda-recipes/pull/53015

Reply:
Good catch!

Reply:
whoops